introspection-engine/introspection-engine-tests/tests/re_introspection/mod.rs

```diff
@@ -1180,9 +1180,15 @@ async fn virtual_cuid_default(api: &TestApi) -> TestResult {
                 t.add_column("id", types::primary());
             });
         })
-        .await?;
+        .await
+        .unwrap();
 
     let input_dm = indoc! {r#"
+        datasource db {
+            provider = "postgres"
+            url = env("TEST_DATABASE_URL")
+        }
+
```

Comment:
Can you use api.datamodel_with_provider here?

Reply:
Done — at least with the equivalent that is in the IE's TestApi